### PR TITLE
Module 1: Initial things branch

### PR DESCRIPTION
**Pull request by bahoz-ali**
**it's like a template project -**

- added HTML file
- added style CSS file 
- added linter
- added gitignore file to ignore some files..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-MICROVERSE
+
+> this is very simple project to learn about pull request and linter and README file...
+
+
```
Comment (tufoinnkuo10) [inline README.md:7]: -[optional] Great job on adding the correct `README` template. please add a brief description of what the project is about.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (tufoinnkuo10) [inline index.html:7]: - [ optional] please try to link your `CSS` file to your `HTML` file.

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```
Comment (bahoz-ali) [inline .github/workflows/linters.yml:8]: thanks its very good job, but you have to add this code `
int a = 5;
int y = a*12; 
if(y>50) console.log("good work")
`

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @bahoz-ali 

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## To highlight 👍
✅ You used the correct Github flow
✅ Good pull request summary description
✅ your project works as expected
✅ linter error checks validated
✅ Good READMe file and well documented
✅ Added HTML and CSS file.
✅ Great use of the gitignore file

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

-------
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (bahoz-ali) [reply index.html:7]: thank you, i fixed it.

File: README.md (hunk 1 above)
Comment (bahoz-ali) [reply README.md:7]: thanks, but I thought it was just a simple project so I did not add a description of the project.

Comment (tufoinnkuo10) [reply README.md:7]: That is why I indicated it is optional. it is just to keep it professional no matter how small a project is. Great job.
Happy coding 😄

**Pull request merged**